api/python/t4/api.py

```diff
@@ -69,7 +70,7 @@ def get(src):
         tuple: ``(data, metadata)``.  Does not work on all objects.
     """
     data, meta = get_bytes(fix_url(src))
-    ext = pathlib.Path(src).suffix
+    ext = pathlib.PurePosixPath(src).suffix
```

Comment:
Here, you actually want to use the result of `fix_url(src)`, which will always use `/`. Also, `urlparse(...).path`. It can have a query string like `versionId=...`.

Reply:
Ah.  I thought we only accepted URLs as input, and didn't realize we were using query strings now.  S3 doesn't, from what I understand, and I didn't figure the filesystem would.  But if we're parsing them for our own use, that makes sense.  I probably should have accounted for the possibility anyways.

Reply:
fixed.